pkg/products/observability/reconciler.go

```diff
@@ -86,6 +120,23 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	productNamespace := r.Config.GetNamespace()
 
 	phase, err := r.ReconcileFinalizer(ctx, client, installation, string(r.Config.GetProductName()), func() (integreatlyv1alpha1.StatusPhase, error) {
+		// Remove redundant finalizer from observability CR
+		oo := &observability.Observability{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "observability-stack",
+				Namespace: operatorNamespace,
+			},
+		}
+		err := client.Get(ctx, k8sclient.ObjectKey{Name: oo.Name, Namespace: oo.Namespace}, oo)
+		if err != nil && !k8serr.IsNotFound(err) {
+			r.log.Error("Failed to get observability CR", err)
+		}
+		oo.SetFinalizers(resources.Remove(oo.GetFinalizers(), "observability-cleanup"))
+		err = client.Update(ctx, oo)
+		if err != nil && !k8serr.IsNotFound(err) {
+			r.log.Error("Failed to remove redundant finalizer from observability CR", err)
+		}
+
```

Comment:
I think this code shouldn't be included? 🤔

Reply:
:thinking:  no it shouldn't , rebase against valerymo/oo-install is not removing it , close this pr and will cherry pick my commits to a new branch